src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -60,6 +60,18 @@ private void AssertNonZeroWindowsZeroUnix(long value)
             }
         }
 
+        private void AssertNonZeroAllZeroDarwin(long value)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
```

Comment:
I'd rather not test OSX since presumably we'll eventually support it and adding test in a sense documents the behavior

Reply:
Ah, I see, this is following the existing pattern.. fine then